Sources/GRPC/ClientConnection.swift

```diff
@@ -500,12 +514,22 @@ extension ChannelPipeline.SynchronousOperations {
     connectionKeepalive: ClientConnectionKeepalive,
     connectionIdleTimeout: TimeAmount,
     httpTargetWindowSize: Int,
+    httpMaxFrameSize: Int,
     errorDelegate: ClientErrorDelegate?,
     logger: Logger
   ) throws {
+    let initialSettings = [
+      // As per the default settings for swift-nio-http2:
```

Comment:
NIOHTTP2 makes its default settings available in a global-scope variable (yeah I know) so you can just extend that array if you'd prefer.

Reply:
Good ol' `nioDefaultSettings`! I'd rather be explicit here, I think.

Just a thought: should we set `maxConcurrentStreams` to `0`? The client doesn't tolerate or expect the server to open streams so explicitly setting it to 100 feels odd.

Reply:
Sure, why not.